core/config/locales/en.yml

```diff
@@ -1138,7 +1138,7 @@ en:
     log_in_to_continue: Log in to continue
     logs: "Logs"
     logged_in_as: Logged in as
-    logged_in_succesfully: Logged in successfully
+    logged_in_successfully: Logged in successfully
```

Comment:
I'm not sure if this will break anything...

Reply:
@vfonic could probably break translations in spree_i18n but that's an easy fix